pkg/controller/util/utils.go

```diff
@@ -52,3 +52,21 @@ func IsNodeAgentReady(c client.Client) bool {
 	}
 	return conf.Status.NodeAgent.Phase == csiv1.NodeAgentPhaseRunning
 }
+
+func Contains(list []string, s string) bool {
+	for _, v := range list {
+		if v == s {
+			return true
+		}
+	}
+	return false
+}
+
+func Remove(list []string, s string) []string {
+	for i, v := range list {
+		if v == s {
+			list = append(list[:i], list[i+1:]...)
+		}
+	}
+	return list
+}
```

Comment:
Doesn't Go offer an implementation for this logic?

Anyway, I'm not sure these functions are relevant to this file.
I suggest extracting them to another file i.e. `pkg/controller/util/datastructures.go`

Reply:
not that I know of.
done